app/views/pensions/annualAllowance/PensionProviderTaxPaidAnnualAllowanceView.scala.html

```diff
@@ -0,0 +1,92 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.templates.Layout
+@import views.html.templates.helpers.InputTextAlt
+@import views.html.templates.helpers.Heading
+@import views.html.templates.helpers.InputRadioAlt
+@import views.html.templates.helpers.Button
+@import views.html.templates.helpers.ErrorSummary
+@import utils.ViewUtils._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(
+        layout: Layout,
+        heading: Heading,
+        button: Button,
+        inputText: InputTextAlt,
+        formWithCsrf: FormWithCSRF,
+        inputRadio: InputRadioAlt,
+        errorSummary: ErrorSummary,
+        govukRadios : GovukRadios,
+        govukInput : GovukInput
+)
+
+@(form: Form[(Boolean, Option[BigDecimal])], taxYear: Int)(implicit request: AuthorisationRequest[_], messages: Messages, appConfig: AppConfig)
+
+@titleForUse = @{messages(s"common.pensions.didYourPensionSchemesAgreeToPayTheTax.title.${if(request.user.isAgent) "agent" else "individual"}")}
+
+@hasFormErrors = @{form.hasErrors}
+
+@AmountInput=@{inputText(
+    form,
+    id="amount-2",
```

Comment:
Why is this field called `amount-2`, when there's only one amount field?

Reply:
I have updated the id to amount-1

Reply:
You have updated the id but name the name.

And given that we only the one, perhaps we could use `amount` for both?

Reply:
In discussing this with @srinivaspach , I understand that it would be rather a lot to take on at the moment.

As such, we'll leave it for now.